StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -10,6 +10,18 @@ import Foundation
 @_spi(STP) public extension PaymentSheet {
     /// 🚧 Under construction
     struct IntentConfiguration {
+
+        /// Creates a `PaymentSheet.IntentConfiguration` with the given values
+        /// - Parameters:
+        ///   - mode: The mode of this intent, either payment or setup
+        ///   - captureMethod: The capture method of this intent, either automatic or manual
+        ///   - paymentMethodTypes: The payment method types for the intent
+        public init(mode: Mode, captureMethod: CaptureMethod, paymentMethodTypes: [String]? = nil) {
```

Comment:
Let's give captureMethod a default value of `.automatic`, it should be very rarely used